src/cmd/singularity/cli/search.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2018, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package cli
+
+import (
+	"github.com/singularityware/singularity/src/docs"
+	"github.com/singularityware/singularity/src/pkg/library/client"
+	"github.com/singularityware/singularity/src/pkg/sylog"
+	"github.com/spf13/cobra"
+)
+
+var (
+	// SearchLibraryURI holds the base URI to a Sylabs library API instance
+	SearchLibraryURI string
+)
+
+func init() {
+	SearchCmd.Flags().SetInterspersed(false)
+	SearchCmd.Flags().StringVar(&SearchLibraryURI, "library", "https://library.sylabs.io", "URI for library to search")
```

Comment:
That is the correct hard-coded default. Right now you have to use the `--library` option to use something local. What's the general strategy for config at present? Should this be settable via a `SINGULARITY_LIBRARY` var. Do we have any kind of structure for a user-local configuration file?

Reply:
There is knowledge of a need to be able to set a library URI, builder URI, use a different token etc. easily. Probably at both the site and the user level. I don't know we have any particulars of how to store that firmed up though. Apologies - I was thinking thoughts on this would come out when ENV var handling was tackled etc, and understand we are pretty far along now to be needing to add this.

Reply:
OK.  Fair enough.  For now hard-coding this seems to be the way to go.